controlpanel/frontend/views/user.py

```diff
@@ -12,6 +13,15 @@
 from controlpanel.api.models import User
 
 
+def three_months_ago():
```

Comment:
```suggestion
def ninety_days_ago():
```

Maybe 90 days makes it clearer how long it actually is?

Reply:
:+1:

Reply:
Done